alibi_detect/od/tests/test__knn/test__knn.py

```diff
@@ -244,7 +260,12 @@ def test_knn_integration():
     result = result['data']['is_outlier'][0]
     assert result
 
-    tsknn = torch.jit.script(knn_detector.backend)
+    ts_knn = torch.jit.script(knn_detector.backend)
     x = torch.tensor([x_inlier[0], x_outlier[0]], dtype=torch.float32)
-    y = tsknn(x)
+    y = ts_knn(x)
+    assert torch.all(y == torch.tensor([False, True]))
+
+    ts_knn.save(tmp_path / 'pca.pt')
```

Comment:
I guess it doesn't make a difference, but should `pca.pt` really be `knn.pt` here? Likewise for Mahalanobis test